### ISSUE TITLE
Audio only

### ISSUE THREAD

**Issue by sloothword** [quasarframework/quasar-ui-qmediaplayer#120]
**Describe the bug**
When setting `type="audio"` the video is still shown.

**To Reproduce**
 ```
<q-media-player type="audio" source="statics/Video.mp4">
```
![image](https://user-images.githubusercontent.com/7378822/80861594-ca35bf80-8c6f-11ea-84f2-f4cf15c103c5.png)

**Expected behavior**
Only the audio of the videosource is played and there is no video playback.

![image](https://user-images.githubusercontent.com/7378822/80861642-1d0f7700-8c70-11ea-95ea-4f43075cb0c4.png)

**Additional context**
I guess the underlying problem is using `<video>` also for audio sources.
When i manually replace the `<video>` with `<audio>` the video playback disappears.

***Possible fix 1***
Use `<audio>` element when `type=audio`.
However, according to the source comment this is intentionally not done:
   ```
 __renderAudio (h) {
      [...]

      // This is on purpose (not using audio tag).
      // The video tag can also play audio and works better if dynamically
      // switching between video and audio on the same component.
      return h('video', {
```
(I want to switch dynamically between video and audio display of the same video file)

***Possible fix 2***
My current workaround is to hide the `<video>` element with css:
`<q-media-player :type="type" :class="{'audio-only': type=='audio'}">`
and 
```
<style>
    .audio-only video {
        display:none
    }
</style>
```
This could be an easy feature to include in the QMediaPlayer itself.
However, i do not know how reliably that works across browsers.

**Comment by hawkeye64** [comment]
@sloothword You have certainly hit upon a few issues I encountered while developing QMediaPlayer.

And certainly, playing just the audio of a video was never one of the use-cases I imagined.

Let me see what I can come up with.

**Comment by hawkeye64** [comment]
@sloothword I have added a fix, with new property `no-video` used in conjunction with `type="audio"` it will for QMediaPlayer to use the `<audio>` element, instead of the `<video>` tag. The example I wrote plays the video with audio only (no video). But, I haven't tested with dynamic changing, so you may or may not get issues. 

Will be released with v1.0.23
